README.md

```diff
@@ -103,35 +87,9 @@ please read "User's Guide".
 
 | Framework  | Model                               | Data Type                             | Batch Sizes                            | Input Size                               |
 |------------|-------------------------------------|---------------------------------------|----------------------------------------|------------------------------------------|
-| JAX        | T5-Large                            | FP32, FP16, BF16                      | 1, 16, 24, 32, 48, 64, 512             | Sequence length 512                      |
```

Comment:
@mariecwhite we can probably remove things that are currently unmaintained. Thoughts?

Reply:
T5-Large is still being benchmarked on server. I can update this list with the workloads that are currently disabled due to an issue and link to the issue.